lib/generators/solidus_subscriptions/install/install_generator.rb

```diff
@@ -6,6 +6,7 @@ class InstallGenerator < Rails::Generators::Base
       source_root File.expand_path('templates', __dir__)
 
       class_option :auto_run_migrations, type: :boolean, default: false
+      class_option :frontend, type: :string, default: 'legacy'
```

Comment:
Using a string instead of a boolean looks better to me, since more things could end up being frontends (api, graphql, etc) and being able to select the right integration seems like a better choice.

Anyways the name is `classic` in the solidus installer, and I'm fine with both fixing it or moving it to a boolean, we can change later if necessary 👍

Reply:
I used this format to follow the [main installer](https://github.com/solidusio/solidus/blob/16d1e3704fbbe630a584c919de1a640a6ab6c6f8/core/lib/generators/solidus/install/install_generator.rb#L49). For sure I should at least change `legacy` to `classic` to follow that convention.  Not sure which approach is best at this point, @elia do you have any thought?

Reply:
I think we posted at the same time, but I'll restate my take: while right now I see using a string as the way forward I'm fine with both. 

_I other words I think we'll have a better understanding on how things "should be done" in a month or two, and we'll be able to codify it somewhere (e.g.  in solidus_dev_support)._

Reply:
Changed to `classic`, but open to change while I wait for @waiting-for-dev opinion.

Reply:
I also thought about the potential for adding new storefronts. However, I see it as a premature optimization, plus we won't add any other new frontend in the near future. Having code that does nothing looks off. We've been discussing whether to call it `classic` or `legacy`, but we could call it `potato`, and it'd make no difference 😆 But I'm not super-against it, as it could help in the future to avoid breaking changes. However, I'd try to be consistent throughout the extension ecosystems, so we should do the same here and in the PayPal one.